CometServer/Configuration/ServiceMessagingConfig.cs

```diff
@@ -0,0 +1,115 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="ServiceMessagingConfig.cs" company="RHEA System S.A.">
+//    Copyright (c) 2015-2023 RHEA System S.A.
+//
+//    Author: Sam Gerené, Alex Vorobiev, Alexander van Delft, Nathanael Smiechowski, Antoine Théate
+//
+//    This file is part of CDP4-COMET Webservices Community Edition. 
+//    The CDP4-COMET Webservices Community Edition is the RHEA implementation of ECSS-E-TM-10-25 Annex A and Annex C.
+//
+//    The CDP4-COMET Webservices Community Edition is free software; you can redistribute it and/or
+//    modify it under the terms of the GNU Affero General Public
+//    License as published by the Free Software Foundation; either
+//    version 3 of the License, or (at your option) any later version.
+//
+//    The CDP4-COMET Webservices Community Edition is distributed in the hope that it will be useful,
+//    but WITHOUT ANY WARRANTY; without even the implied warranty of
+//    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+//    GNU Affero General Public License for more details.
+//
+//    You should have received a copy of the GNU Affero General Public License
+//    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+// </copyright>
+// --------------------------------------------------------------------------------------------------------------------
+
+namespace CometServer.Configuration
+{
+    using Microsoft.Extensions.Configuration;
+
+    /// <summary>
+    /// The Inter service messaging configuration.
+    /// </summary>
+    public class ServiceMessagingConfig
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ServiceMessagingConfig"/> class.
+        /// </summary>
+        public ServiceMessagingConfig()
+        {
+            // set defaults
+            this.IsEnabled = true;
```

Comment:
I don't think that hard coding stuff here is what we want. Either we have a settings file, or we don;t and then it should just does not work (and log why it doesn't work).